Source/MediaInfo/Image/File_Jpeg.cpp

```diff
@@ -1189,6 +1226,196 @@ void File_Jpeg::APP1_EXIF()
+        Element_Begin1("Profile version number");
+            Info_B1(M,                                          "Major");
+            BS_Begin();
+            Info_S1(4, m,                                       "Minor");
+            Info_S1(4, f,                                       "Fix");
+            BS_End();
+            Skip_B2(                                            "Reserved");
+            Element_Info1(Ztring().From_Number(M)+__T('.')+Ztring().From_Number(m)+__T('.')+Ztring().From_Number(f));
+        Element_End0();
+        Skip_C4(                                                "Profile/Device class");
+        Get_C4 (ColorSpace,                                     "Colour space of data");
+        Skip_C4(                                                "PCS");
+        Element_Begin1("Date/Time");
+            Info_B2(YY,                                         "Year");
+            Info_B2(MM,                                         "Month");
+            Info_B2(DD,                                         "Day");
+            Info_B2(hh,                                         "Hour");
+            Info_B2(mm,                                         "Minute");
+            Info_B2(ss,                                         "Second");
+            #if MEDIAINFO_TRACE
+                string DateTime;
+                DateTime+='0'+YY/1000;
+                DateTime+='0'+(YY%1000)/100;
+                DateTime+='0'+(YY%100)/10;
+                DateTime+='0'+(YY%1000);
+                DateTime+='-';
+                DateTime+='0'+MM/10;
+                DateTime+='0'+(MM%10);
+                DateTime+='-';
+                DateTime+='0'+DD/10;
+                DateTime+='0'+(DD%10);
+                DateTime+=' ';
+                DateTime+='0'+hh/10;
+                DateTime+='0'+(hh%10);
+                DateTime+=':';
+                DateTime+='0'+mm/10;
+                DateTime+='0'+(mm%10);
+                DateTime+=':';
+                DateTime+='0'+ss/10;
+                DateTime+='0'+(ss%10);
+                Element_Info1(DateTime.c_str());
+            #endif //MEDIAINFO_TRACE
+        Element_End0();
+        Skip_C4(                                                "'acsp' profile file signature ");
+        Skip_C4(                                                "Primary platform signature");
+        Skip_B4(                                                "Profile flags");
+        Skip_C4(                                                "Device manufacturer");
+        Skip_B4(                                                "Device model");
+        Skip_B4(                                                "Device attributes TODO 1");
+        Skip_B4(                                                "Device attributes TODO 2");
+        Skip_B4(                                                "Rendering Intent ");
+        Element_Begin1("nCIEXYZ values");

```

Comment:
from http://www.color.org/icc1V42.pdf "XYZ values" would be better

Reply:
Spec say "The nCIEXYZ values of the illuminant of the PCS"
Both look wrong :).
Will change to "Illuminant of the PCS"

Reply:
" from http://www.color.org/icc1V42.pdf ": it is [deprecated by v4.3](http://www.color.org/icc_specs2.xalter)